api/client/create.go

```diff
@@ -94,20 +93,37 @@ func (cli *DockerCli) createContainer(config *runconfig.Config, hostConfig *runc
 		defer containerIDFile.Close()
 	}
 
+	repo, tag := parsers.ParseRepositoryTag(config.Image)
+	if tag == "" {
+		tag = tags.DEFAULTTAG
+	}
+	ref := registry.ParseReference(tag)
+	var trustedRef registry.Reference
+
+	if isTrusted(repo) && !ref.IsTrusted() {
```

Comment:
shouldnt there be a TODO for when isTrusted(repo) is false we will eventually want to explode and die with 80 bajillion warnings